docs/debugging-tips.adoc

```diff
@@ -147,11 +57,13 @@ Fetch both of these down. In wireshark preferences set ssl.keylog_file to point
 
 aktualizr can be tested against a dummy repository containing fake images
 
-First, generate a repository using `aktualizr-repo` command:
+First, generate a repository using https://github.com/advancedtelematic/aktualizr/blob/master/src/aktualizr_repo/main.cc[aktualizr-repo] tool:
 
-    `aktualizr-repo generate repo_dir`
+```
+aktualizr-repo generate <repo_dir>
+```
```

Comment:
maybe makes sense to add steps required to generate a new update(s)/images and trigger their updates:

1. aktualizr-repo image
2. aktualizr-repo addtarget
3. aktualizr-repo signtargets

or reference to this doc https://github.com/advancedtelematic/aktualizr/blob/master/docs/aktualizr-repo.adoc

Reply:
Good call. I added a reference to that doc. No need to duplicate the content. I also made all the links relative where possible.